### PR TITLE
[DQM] DT Fixing FEDFatal

### PR DESCRIPTION
**Pull request by jfernan2**
#### PR description:

This PR fixes DT DQM FEDFatal plot setting a Minumum number of Links/wheel with notOKFlag or TDC fatal errors to consider a FEDfatal event. The proposed threshold value by experts for such events is 15 Links/Wheel

In addtion, this PR:
- Fixes labels and filling to follow DT uROS/FED mapping
- Removes obsolete code
- Brings Online and Offline DTDataIntegirty code into one single file, for future easier maintenance, DQM/DTMonitorModule/interface/DTDataIntegrityUrosOffline.h class is not needed any longer

Some small changes in DT DQM plots are expected

#### PR validation:

Tested that Online DT DQM client works as expected and runTheMatrix.py -l limited works even in perLSsaving DQM mode

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

NO backport needed

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

File: DQM/DTMonitorModule/interface/DTDataIntegrityTask.h [hunk 1]
```diff
@@ -15,7 +15,10 @@
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
 #include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
```
Comment (perrotta) [inline DQM/DTMonitorModule/interface/DTDataIntegrityTask.h:21]: Is this `include` needed here?

File: DQM/DTMonitorModule/interface/DTDataIntegrityTask.h [hunk 2]
```diff
@@ -15,7 +15,10 @@
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
 #include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
```
Comment (perrotta) [inline DQM/DTMonitorModule/interface/DTDataIntegrityTask.h:20]: Is this `include` needed here?

File: DQM/DTMonitorModule/src/DTDataIntegrityTask.cc [hunk 3]
```diff
@@ -603,86 +733,92 @@ void DTDataIntegrityTask::processuROS(DTuROSROSData& data, int fed, int uRos) {
...
+      int sector = link + 1;
+      errorX[5][link][wheel + 2] += 1;
+      if (mode != 1) {
+        if (sector == 9)
+          sector = 10;
+        ROSSummary->Fill(tdcError_ROSSummary, sector - 1);  //link 0 = ROS 1
+        int ch25 = 24;
+        if (mode <= 2) {
+          urosHistos[(uROSError)*1000 + (wheel + 2) * 100 + (sector - 1)]->Fill(tdcError_ROSError, ch25);
+          if (mode <= 1)
```
Comment (perrotta) [inline DQM/DTMonitorModule/src/DTDataIntegrityTask.cc:745]: Given L738 above, this should be 
```suggestion
          if (mode < 1)
```

**Comment by jfernan2** [comment]
Thanks @perrotta 
I have implemented your comments

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**